.github/workflows/discourse-plugin.yml

```diff
@@ -51,26 +55,12 @@ jobs:
             echo "manager=${{ env.JS_PKG_MANAGER_NULL_VALUE }}" >> $GITHUB_OUTPUT
           fi
 
-      - name: Install package manager
-        if: steps.js-pkg-manager.outputs.manager != env.JS_PKG_MANAGER_NULL_VALUE
-        run: npm install -g ${{ steps.js-pkg-manager.outputs.manager }}
-
-      - name: Set up Node.js
-        if: steps.js-pkg-manager.outputs.manager != env.JS_PKG_MANAGER_NULL_VALUE
-        uses: actions/setup-node@v4
-        with:
-          node-version: 22
-          cache: ${{ steps.js-pkg-manager.outputs.manager }}
```

Comment:
`yarn`, `pnpm` and `node` are already installed in the test image.

Reply:
By default, the `discourse/discourse_test:slim` image will be used. On our self hosted runners, we can optimize things by using the `discourse/discourse_test:release` image which is already present.